exports/scripts/find.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+
+"""See help fields inside _main or run find.py --help for information."""
+
+from __future__ import annotations
+
+import argparse
+import sys
+from fnmatch import fnmatchcase
+from pathlib import Path
+
+
+def _glob_paths(root_path: Path, patterns: list[str]) -> set[Path]:
+    paths: set[Path] = set()
+    for p in patterns:
```

Comment:
How about `pattern` instead of `p`? Just to differentiate it from `path` on the next line.